theming/theme-package.rst

```diff
@@ -536,14 +532,15 @@ Replace the placeholder with the real Plone top-navigation links:
      css:theme-children=".navbar-nav"
      css:content-children=".plone-navbar-nav" />
 
-Here we take the list of links from Plone and replace the placeholder links in the theme with it.
+Here we take the list of links from Plone and replace the placeholder links in
+the theme with it.
```

Comment:
I know there are lots of places here where we don't follow it, but the rule for documentation is to use semantic linefeeds, not PEP8.  http://docs.plone.org/4/en/about/styleguide.html#id2